packages/common/src/invitationCode.test.ts

```diff
@@ -18,7 +18,15 @@ describe('Invitation code helper', () => {
     expect(invitationDeepUrl('validCode')).toEqual('quiet://?code=validCode')
   })
 
+  it('builds proper invitation deep url with empty code', () => {
+    expect(invitationDeepUrl()).toEqual('quiet://?code=')
```

Comment:
Invitation URL (both share url and deep url) is being changed in another PR

Reply:
Oh! I was not aware, I will close this PR then 👍